src/ServiceControl.Audit.AcceptanceTests/AcceptanceTest.cs

```diff
@@ -63,52 +61,33 @@ public async Task Setup()
             Trace.Listeners.Add(textWriterTraceListener);
 
             TransportIntegration = (ITransportIntegration)TestSuiteConstraints.Current.CreateTransportConfiguration();
-            DataStoreConfiguration = TestSuiteConstraints.Current.CreateDataStoreConfiguration();
 
-            if (DataStoreConfiguration.DataStoreTypeName == nameof(DataStoreType.SqlDb))
-            {
-                await ResetSqlDb(DataStoreConfiguration.ConnectionString).ConfigureAwait(false);
-            }
+            StorageConfiguration = new AcceptanceTestStorageConfiguration();
+
+            await StorageConfiguration.Configure();
 
             var shouldBeRunOnAllTransports = GetType().GetCustomAttributes(typeof(RunOnAllTransportsAttribute), true).Any();
             if (!shouldBeRunOnAllTransports && TransportIntegration.Name != "Learning")
             {
                 Assert.Inconclusive($"Not flagged with [RunOnAllTransports] therefore skipping this test with '{TransportIntegration.Name}'");
             }
 
-            var shouldBeRunOnAllDataStores = GetType().GetCustomAttributes(typeof(RunOnAllDataStoresAttribute), true).Any();
-            if (!shouldBeRunOnAllDataStores && DataStoreConfiguration.DataStoreTypeName != nameof(DataStoreType.RavenDb))
-            {
-                Assert.Inconclusive($"Not flagged with [RunOnAllDataStores] therefore skipping this test with '{DataStoreConfiguration.DataStoreTypeName}'");
-            }
```

Comment:
What was the reason to drop this attribute? This means that all acceptance tests will now run on SQL

Reply:
There is no sql acceptance tests running for SQL since we don't have a project yet. And once we do there is no need for this attribute since we would instead exclude the actual tests from the new project?